apps/site/assets/ts/models/alert.ts

```diff
@@ -10,3 +10,47 @@ export const isDiversion = ({ effect }: Alert): boolean =>
   effect === "stop_closure" ||
   effect === "station_closure" ||
   effect === "detour";
+
+const withLeadingZero = (n: string): string => `0${n}`.slice(-2);
+
+const activePeriodToDates = (
+  activePeriod: TimePeriodPairs
+): (Date | null)[] => {
+  const datePattern = /^(\d{4})-(\d{1,2})-(\d{1,2})\s(\d{1,2}):(\d{2})$/;
+
+  return activePeriod.map(
+    (d: string): Date | null => {
+      const match = datePattern.exec(d);
+      if (match) {
+        const [, year, rawMonth, rawDay, rawHour, min] = match;
+        return new Date(
+          `${year}-${withLeadingZero(rawMonth)}-${withLeadingZero(
+            rawDay
+          )}T${withLeadingZero(rawHour)}:${min}:00`
+        );
+      }
+      return null;
+    }
+  );
+};
+
+const isCurrentLifecycle = ({ lifecycle }: Alert): boolean =>
+  lifecycle === "new" ||
```

Comment:
This is good. `["new", "ongoing", "ongoing_upcoming"].includes(lifecycle)` also works.